<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_up</class>
 <widget class="QMainWindow" name="sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>401</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Century Schoolbook&quot;;
font: 13pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>sign up</string>
    </property>
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0, 0, 0, 0);
border:2px solid rgb(33, 180, 255);
border-bottom : 2px solid rgb(33, 150, 243);
border-radius: 20px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leEmail">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0, 0, 0, 0);
border:2px solid rgb(33, 180, 255);
border-bottom : 2px solid rgb(33, 150, 243);
border-radius: 20px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>480</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border:2px solid rgb(33, 180, 255);
border-bottom : 2px solid rgb(33, 150, 243);
color: rgb(0, 0, 0);
border-radius: 25px;</string>
     </property>
     <property name="text">
      <string>sign up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leUsername">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0, 0, 0, 0);
border:2px solid rgb(33, 180, 255);
border-bottom : 2px solid rgb(33, 150, 243);
border-radius: 20px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePassword">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0, 0, 0, 0);
border:2px solid rgb(33, 180, 255);
border-bottom : 2px solid rgb(33, 150, 243);
border-radius: 20px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>e_mail address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
color:rgb(33, 180, 255);
background-color: rgb(0, 0, 0,0);</string>
     </property>
     <property name="text">
      <string>_______________________________</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>leEmail</zorder>
    <zorder>pushButton</zorder>
    <zorder>leName</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>leUsername</zorder>
    <zorder>lePassword</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;
image: url(:/img/54907784c3b82319fa78bb57df0e90c3.png);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(33, 180, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1353</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
